Chisel/Makefile

```diff
@@ -1,7 +1,7 @@
 PREFIX ?= /usr/local/lib
 
-export INSTALL_NAME = ""
-ifneq ($(LD_DYLIB_INSTALL_NAME), "")
+export INSTALL_NAME =
+ifneq ($(LD_DYLIB_INSTALL_NAME),)
```

Comment:
@keith does this change look legit? The original here was still adding `LD_DYLIB_INSTALL_NAME=...` to `xcodebuild` command, but this change appears to fix that.

Reply:
yea seems fine, shouldn't have had the quotes in the first place I think, my bad.